app/src/main/java/com/sun/funnytoeic/di/RepositoryModule.kt

```diff
@@ -1,6 +1,11 @@
 package com.sun.funnytoeic.di
 
+import com.sun.funnytoeic.data.local.AppDatabase
+import com.sun.funnytoeic.data.repository.impl.VocabularyRepositoryImpl
 import org.koin.dsl.module
 
 val repositoryModule = module {
+    single { VocabularyRepositoryImpl(get()) }
```

Comment:
```suggestion
    single< VocabularyRepository> { VocabularyRepositoryImpl(get()) }
```
Để chắc chắn các module tuơng tác vs nhau qua interface 
em có thể viết như trên